src/lib/react/React.hx

```diff
@@ -87,3 +93,9 @@ extern interface ReactChildren
 }
 
 typedef CreateElementType = haxe.extern.EitherType<haxe.extern.EitherType<String, haxe.Constraints.Function>, Class<ReactComponent>>;
+
+typedef ReactContext<T> = {
+	var displayName:String;
```

Comment:
Is `displayName` on `ReactContext` a thing? I don't see it in sources

Reply:
According to this https://reactjs.org/docs/context.html#contextdisplayname.

Reply:
I see, they're annoying with all this reflection :D Pretty much anything can get a `displayName` attribute, I guess.